src/js/disability-benefits/containers/YourClaimsPage.jsx

```diff
@@ -45,6 +63,10 @@ class YourClaimsPage extends React.Component {
     }
     return undefined;
   }
+  handleSort(sortObject) {
+    this.props.sortClaims(sortObject.value);
+    this.changePage(1);
+  }
   changePage(page) {
     this.props.changePage(page);
     scrollToTop();
```

Comment:
This scrollToTop call feels annoying when you're just changing the sorting dropdown. We could also skip it when changing tabs.